data/tests/images.ts

```diff
@@ -1,11 +1,13 @@
+import { equalTo0, equalTo1 } from './pass'
+
 import { Test } from '../../types'
-import { eq1 } from './pass'
+import { filterCaseRatings } from './queryAugmentation'
 
 const tests: Test[] = [
   {
-    label: 'Precision',
+    label: 'precision',
```

Comment:
Any reason to lowercase there? Aesthetically I like title case, less ++programmeRx10

Reply:
Yeah, your comment ☝️ is right - they're lowercased to make the combination of this label with the namespace look a bit nicer. I like the idea of eg `Precision in images` though. Will make that change now